src/test/scala/zio/sqs/producer/ProducerSpec.scala

```diff
@@ -579,9 +610,11 @@ object ProducerSpec extends DefaultRunnableSpec {
   override def aspects: List[TestAspect[Nothing, TestEnvironment, Nothing, Any]] =
     List(TestAspect.executionStrategy(ExecutionStrategy.Sequential))
 
-  def queueResource(capacity: Int): Task[ZManaged[Any, Throwable, Queue[SqsRequestEntry[String]]]] =
+  def queueResource(capacity: Int): Task[ZIO[Any, Throwable, Queue[SqsRequestEntry[String]]]] =
     Task.succeed {
-      Queue.bounded[SqsRequestEntry[String]](capacity).toManagedWith(_.shutdown)
+      ZIO.scoped {
```

Comment:
This one's wrong too, isn't it?

Reply:
Also not sure why it was nested in the first place. The outer `Task` doesn't seem necessary.